integration_tests/profiles.yml

```diff
@@ -1,5 +1,5 @@
 integration_tests:
-  target: bigquery
+  target: snowflake
```

Comment:
How does the target work here, now when we have both bigquery and snowflake tests in the outputs? 🤔

Reply:
target here is the default target.

which means by default we will upload to bigquery, unless running `dbt run --target snowflake`